### ISSUE TITLE
NASNet-Mobile is ported

### ISSUE THREAD

**Issue by veronikayurchuk** [pytorch/vision#454]
Hi!
NASNet-A-Mobile has been successfully ported to PyTorch!
https://github.com/veronikayurchuk/pretrained-models.pytorch
The results are:
**Acc@1 74.080, acc@5 91.740**

Hope, it would be useful!

**Comment by fmassa** [comment]
Nice! Great work!
At some point it might be good to have those in torchvision, if we manage to retrain them from scratch (in order to keep the normalizations and preprocessing consistent with the other models in torchvision), but that might require quite some work to get those working as the official ones.
